mod.ts

```diff
@@ -40,9 +43,6 @@ async function redirect(url: string, fetchTags: FetchTags): Promise<string> {
       const tag = tags[tags.length - 1];
       return `https://deno.land/${x}${name}@${tag}${path}`;
     }
-    if(range === "latest") {
-      return `https://deno.land/${x}${name}${path}`;
-    }
```

Comment:
In the event that a tag bearing the literally name "latest" exists, it is imperative that we redirect to the corresponding "latest" branch. Although this is an infrequent scenario in practice, it remains theoretically plausible and warrants due consideration. So we should take care of them. Please do not change the order.

Reply:
I see your concern, I will reverse this and submit another PR.